src/com/from/zenhub/IfNumberIsPositiveOrNegative.java

```diff
@@ -0,0 +1,21 @@
+package com.from.zenhub;
+
+import java.util.Scanner;
+
+public class IfNumberIsPositiveOrNegative {
+
+	public static void main(String[] args) {
+		Scanner scanner = new Scanner(System.in);
+		System.out.println("Give any integer: ");
+		int x = scanner.nextInt();
+		
+		if ( x > 0){
```

Comment:
correct the spacing at different part of this this code. One example is remove space between ( and x